app/api/schema/event_statistics.py

```diff
@@ -50,8 +51,23 @@ def sessions_pending_count(self, obj):
     def sessions_rejected_count(self, obj):
         return Session.query.filter_by(event_id=obj.id, state='rejected').count()
 
+    def speakers_count_type(self, obj, state='pending'):
```

Comment:
@uds5501 why is the default state pending?
wouldn't it be better to not have default as pending, rather it's better if an error is thrown if state is not supplied as the function is basically returning sessions by states? Pending seems arbitrary.

Reply:
@CosmicCoder96 Actually I think this function isn't being used anywhere at all so I will be removing it in the next commit